### ISSUE TITLE
{silent: true} bug?

### ISSUE THREAD

**Issue by seantw** [t1m0n/air-datepicker#649]
The **onSelect** event handler is bypassed when calling ` selectDate(date,{silent: true})` , whereas **onBeforeSelect** is still invoked.

**Comment by t1m0n** [comment]
Yep, it should not be triggered, you're right. Thanks for the bug report
